meson.build

```diff
@@ -1,10 +1,13 @@
 project( 'lumberjack' 
   , 'cpp'
-  , version : '0.0.1.-1'
+  , version : '"0.0.1.-1"'
```

Comment:
Quotes is not a typical value for a project version.

Reply:
@eli-schwartz Wow, thanks for the feedback. I didn't realize there were any outside eyes on this project and I'm glad you find it interesting. I'm learning meson on the fly, and I clearly have a long way to go. I'll make sure to implement this change.